content/doc/administration/requirements/windows.adoc

```diff
@@ -0,0 +1,88 @@
+---
+layout: documentation
+title:  Windows Support Policy
+---
+
+This page documents the Windows support policy for the Jenkins server and agents.
+
+== Scope
+
+Jenkins plugins, e.g. plugin:windows-slaves[WMI Windows Agents],
+may set additional requirements to Windows versions on masters and/or agents.
+This page does not document such requirements,
+please refer to plugin documentation.
+
+== Why?
+
+Theoretically Jenkins can run everywhere where you can run a supported Java version,
+but there are some limitations in practice.
+Jenkins core and some plugins include native code or depend on Windows API and subsystems,
+and hence they rely on specific Windows platforms and versions.
+In the case of Windows services we also use link:https://github.com/winsw/winsw[Windows Service Wrapper (WinSW)] which requires .NET Framework.
+
+== Support levels
+
+We define multiple support levels for Windows platforms.
+
+[width="100%",cols="20%,40%,40%",options="header",]
+|===
+|Support level |Description |Platforms
+
+| **Level 1** - Full support
+| We run automated testing for these platforms, and we intend to timely fix the reported issues.
+a| * 64-bit versions of latest Windows and Windows Server versions,
+    with the latest GA update pack
+  * Windows versions used in the official Docker images
+
+| **Level 2** - Supported
+| We do not actively test it, but we intend to keep compatibility.
+  We are happy to accept patches.
+a|
+  * 64-bit Windows 10 versions generally supported by Microsoft
+  * 64-bit Windows Server versions generally supported by Microsoft
+
+| **Level 3** - Best effort
+| Support may have limitations and extra requirements.
+  We do not test compatibility, and we may drop support if there is a need.
+  We will consider patches if they do not put Level 1/2 support at risk and if they do not create bug maintenance overhead.
+a|
+  * 64-bit Windows versions no longer supported by Microsoft
+  * x86 and other non-amd64 architectures
+  * Non-mainstream versions, e.g. Windows Embedded
+  * Preview releases
+  * Windows API emulation engines, e.g. Wine or ReactOS
```

Comment:
why?

Reply:
I do not understand the question

Reply:
why are wine and other emulators level 3.  
to be honest levels  3 and 4 confuse me.
it's just  un supported.
if we could support xp with a single line of code this document says we wouldn't accept it.  but an emulator is fair game whose APIs are always playing  catch-up and are not the same as windows...

Reply:
Emulators might be newer... or not. I am happy to relax the condition in Level 4 to state we may accept patches.
For me "Level 4" is Level 3 which is known to not work